infra/lambda/datasets-endpoint/conftest.py

```diff
@@ -0,0 +1,111 @@
+"""
+Pytest configuration file.
+"""
+
+import logging
+
+import boto3
+import pytest
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def db_prepare(table_name="datasets"):
+    """
+    Prepare DB with some dataset records and clean it up after test is
+    finished.
+    """
+
+    logger.info("Running DB Setup")
+
+    DYNAMODB = boto3.client("dynamodb")
+    DYNAMODB.batch_write_item(
+        RequestItems={
+            table_name: [
+                {
+                    "PutRequest": {
+                        "Item": {
+                            "pk": {
+                                "S": "DATASET#111abc",
```

Comment:
Why is the data type part of the request?

Reply:
Do you mean why we have `DATASET` string part of `pk` value ? If this is the case, it is because `pk` attribute is going to store different kinds of values . Can store dataset id - in this case value will be `DATASET#111abc` or can store dataset version id - will be `VERSION#111abc` .

Reply:
Sorry, I meant the "S" part of that request. Looks like this is a [DynamoDB pattern](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_AttributeValue.html).